### PR TITLE
Dev Test Labs - Updated code sample

### PR DESCRIPTION
**Pull request by jawache**

### CODE REVIEW

File: docs-ref-services/devtest-labs.md [hunk 1]
```diff
@@ -13,22 +13,42 @@ ms.devlang: nodejs
...
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const DevTestLabsClient = require('azure-arm-devtestlabs');
+
+const subscriptionId = 'your-subscription-id';
+const resourceGroupName = 'your-resource-group-name';
+const labName = 'your-lab-name';
+
+msRestAzure.interactiveLogin((err, credentials) => {
+  const client = new DevTestLabsClient(credentials, subscriptionId);
```
Comment (simonaco) [inline docs-ref-services/devtest-labs.md]: We should be using promises here similar to https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/91/commits/8c76d0e358b3e4de15b2197cc8ccdd3933baecf3

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/devtest-labs.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-10/docs-ref-services/devtest-labs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C17%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707171822406033-99%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/devtest-labs.md (hunk 1 above)
Comment (jawache) [inline docs-ref-services/devtest-labs.md]: I missed that thanks, will update :)

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/devtest-labs.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-10/docs-ref-services/devtest-labs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C17%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707171834417997-99%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/devtest-labs.md [hunk 2]
```diff
@@ -13,22 +13,44 @@ ms.devlang: nodejs
...
 ## Overview
 Azure DevTest Labs is a service that helps developers and testers quickly create environments in Azure while minimizing waste and controlling cost. You can test the latest version of your application by quickly provisioning Windows and Linux environments using reusable templates and artifacts. Easily integrate your deployment pipeline with DevTest Labs to provision on-demand environments. Scale up your load testing by provisioning multiple test agents, and create pre-provisioned environments for training and demos.
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure DevTest Labs modules for Node.js
-
-### Management
+### Install npm module
```
Comment (johnpapa) [inline docs-ref-services/devtest-labs.md]: module --> package

and add a sentence here

> Install the DevTest Labs module for Node.js

File: docs-ref-services/devtest-labs.md [hunk 3]
```diff
@@ -13,22 +13,44 @@ ms.devlang: nodejs
...
+  .then(lab => {
+    console.log('Details of lab:');
+    console.dir(lab, { depth: null, colors: true });
+  });
+
+
+```
 
-##Samples
+### Samples
```
Comment (johnpapa) [inline docs-ref-services/devtest-labs.md]: Samples should be 2 ##s, as they apply to all samples here

**Comment by johnpapa** [comment]
needs review before #sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/devtest-labs.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-10/docs-ref-services/devtest-labs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180056429185-99%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/devtest-labs.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-10/docs-ref-services/devtest-labs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180057149565-99%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/devtest-labs.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-10/docs-ref-services/devtest-labs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180408466650-99%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
LGTM

#sign-off

**Pull request merged**